File: php/car.php
<?php
require_once('account.php');
class Car
{
    // Declaración de una propiedad
    public $id = int;
    public $driver = string;
    public $license = string;
    public $passenger = int;

    // Declaración de un método
    public function __construct($driver, $license)
    {
        $this->driver = $driver;
        $this->license = $license;
    }

    public function printData() {
        echo "Licencia: $this->license Driver: ".$this->driver.name;
    }
}
?>

File: java/car.java
class Car {
    private int id;
    private String license;
    private Account driver;
    protected Integer passenger;
    
    public Car(String license, Account driver) {
        this.license = license;
        this.driver = driver;
    }

    void printDataCar() {
        if(passenger != null) {
            System.out.println("License: " + license + " Driver: " + driver.name + " Passengers: " + passenger);
        }
        }

    public int getPassengers() {//getter
        return passenger;
    }

    public void setPassengers(int passenger) {//setter
        if(passenger == 4) {
            this.passenger = passenger;
        }else{
            System.out.println("necesitas asignar 4 pasajeros");
        }
             
    }

    public int getId() {
        return id;
    }

    public void setId(int id) {
        this.id = id;
    }

    public String getLicense() {
        return license;
    }

    public void setLicense(String license) {
        this.license = license;
    }

    public Account getDriver() {
        return driver;
    }

    public void setDriver(Account driver) {
        this.driver = driver;
    }

    
}
